test/integration/targets/virt_net/aliases

```diff
@@ -0,0 +1,5 @@
+destructive
+shippable/posix/group1
```

Comment:
Unless this test works under docker without privileged mode, remove this and use `unsupported`.

Tests that require privileged mode under docker are not run on Shippable.

Reply:
ok, thanks for the clarification.

Reply:
Sorry, I gave you bad instructions.

I forgot that this would run on our RHEL VMs, so having it in `shippable/posix/group1` is appropriate.

Reply:
np, I will push an update. I understand I also need to remove `unsupported`.